src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
The Dom code actually assumes that the first view has a parent..  they both have to be children of the collection, but technically I guess at the moment the 2nd could be detached?  that gets a little weird.  Guess I could check they're both `_isAttached`..